SCF/Assignment2A/HexCalc.java

```diff
@@ -0,0 +1,444 @@
+		
+		//first checking if length is equal
+		if(firstOperand.length() == secondOperand.length())
+		{
+			
+			//if length equal than comparing each character 
+			for(int charIndex = 0; charIndex < firstOperand.length(); charIndex++)
+			{
+				char firstOperandExtractedChar = firstOperand.charAt(charIndex);
+				char secondOperandExtractedChar = secondOperand.charAt(charIndex);
+				
+				//taking numeric value of character and comparing them 
+				if(Character.getNumericValue(firstOperandExtractedChar)!=Character.getNumericValue(secondOperandExtractedChar))
+				{
+					return false;
+				}
+			}
+			return true;
+		}
+		else
+		{
+			return false;
+		}
+	}
+	
+	/**
+	 * 
+	 * Assuming firstOperand is greater 
+	 * @param firstOperand operand for greater than operation
+	 * @param secondOperand operand for greater than operation
+	 * @return true if firstOperand is greater
+	 * @throws IOException if parameter not contain legal value 
+	 */
+	public boolean greaterThanOperation(String firstOperand, String secondOperand) throws IOException
+	{
+		/**
+		 * 
+		 * validating parameter
+		 */
+		if(!(inputValidation(firstOperand) && inputValidation(secondOperand)))
+		{
+			throw new IOException("Invalid Input!!!");
+		}
+		//remove initial zeros 
+		firstOperand = removeInitialZeros(firstOperand);
+		secondOperand = removeInitialZeros(secondOperand);
+		
+		//first checking if length is equal
+		if(firstOperand.length() > secondOperand.length())
+		{
+			return true;
+		}
+		
+		if(firstOperand.length() == secondOperand.length())
+		{
+
+			//if length equal than comparing each character 
+			for(int charIndex=0; charIndex < firstOperand.length(); charIndex++)
+			{
+				char firstOperandExtractedChar = firstOperand.charAt(charIndex);
+				char secondOperandExtractedChar = secondOperand.charAt(charIndex);
+				//taking numeric value of character and comparing them 
+				if(Character.getNumericValue(firstOperandExtractedChar)>Character.getNumericValue(secondOperandExtractedChar))
+				{
+					return true;
+				}
+			}
+			return false;
+		}
+		else
+		{
+			return false;
+		}
+	}
+	
+	/**
+	 * 
+	 * assuming first operand is lesser
+	 * @param firstOperand operand for less than operation
+	 * @param secondOperand operand for less than operation
+	 * @return true if firstOperand is lesser
+	 * @throws IOException if parameter not contain legal value
+	 */
+	public boolean lessThanOperation(String firstOperand, String secondOperand) throws IOException
+	{
+		/**
+		 * validating parameter
+		 */
+		if(!(inputValidation(firstOperand) && inputValidation(secondOperand)))
+		{
+			throw new IOException("Invalid Input!!!");
+		}
+		//remove initial zeros 
+		firstOperand = removeInitialZeros(firstOperand);
+		secondOperand = removeInitialZeros(secondOperand);
+		//first checking if length is equal
+		if(firstOperand.length()<secondOperand.length())
+		{
+			return true;
+		}
+
+		if(firstOperand.length()==secondOperand.length())
+		{
+
```

Comment:
remove extra line from here and from similar places.

Reply:
Done